prac_05/word_occurences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurences
+estimate: 30 mins
+Actual: 40 mins
+"""
+
+text = input("Text: ").lower()
+words = text.split()
+word_occurences = {}
+# initialize the longest width to the width to the first word for comparison
+width = len(words[0])
```

Comment:
width is a bit confusing so I suggest changing the variable name to longest_word_length to make it more clear